### ISSUE TITLE
so, i'm guessing it's not compatible with 2.93 yet? 

### ISSUE THREAD

**Issue by lashmanRB** [leomoon-studios/leomoon-lightstudio#39]
because this is what i'm getting when i try to enable it: 

```Traceback (most recent call last):
  File "D:\blender k-cycles\2.93\scripts\modules\addon_utils.py", line 386, in enable
    mod.register()
  File "C:\Users\username\AppData\Roaming\Blender Foundation\Blender\2.93\scripts\addons\leomoon-lightstudio-master\__init__.py", line 32, in register
    auto_load.register()
  File "C:\Users\username\AppData\Roaming\Blender Foundation\Blender\2.93\scripts\addons\leomoon-lightstudio-master\auto_load.py", line 36, in register
    bpy.utils.register_class(cls)
ValueError: register_class(...): already registered as a subclass 'LLS_PT_Studio'```

**Comment by Prototype-Zer0** [comment]
I wrote a quick fix for it, the API was changed recently but that part of the code is unstable so not sure why it was merged to master, either way here is my PR with the fix, tested it and worked just fine but someone else smarter than me might have better fix: https://github.com/leomoon-studios/leomoon-lightstudio/pull/40
For more information about the API changes: https://wiki.blender.org/wiki/Reference/Release_Notes/2.93/Python_API

**Comment by leomoon** [comment]
This issue should be fixed with LightStudio version 2.8.1:
https://github.com/leomoon-studios/leomoon-lightstudio/archive/refs/heads/master.zip
